RHTE-LAB.adoc

```diff
@@ -547,13 +547,17 @@ Resolving deltas: 100% (412/412), done.
 +
 [subs=+quotes]
 --------------------------------------------------------------------------------
-$ **helm template eunomia/deploy/helm/operator/ \
-  --set eunomia.operator.openshift.route.enabled=true \
-  --set eunomia.operator.image.tag=v0.0.1 \
-  --set eunomia.operator.image.pullPolicy=IfNotPresent | oc apply -f -**
-configmap/eunomia-templates created
+$ **helm template deploy/helm/eunomia-operator/ \
+  --set openshift.route.enabled=true | oc apply -f -**
```

Comment:
@jkupferer this looks like we are reverting back to a `--set` flag that doesn't work. According to https://github.com/KohlsTechnology/eunomia/blob/master/deploy/helm/eunomia-operator/values.yaml, this needs to be set back to `--set eunomia.operator.openshift.route.enabled=true`

Reply:
@etsauer Good catch, fixed. Looks like the README in the eunomia repo is incorrect.